resources/assets/css/style.css

```diff
@@ -206,4 +206,6 @@ input.add-subject-input {font-size: 13px;padding: 4px 12px;}
 .notifications-overlay:after{margin: 19px -34px;bottom: 0;top: initial;}
 .notifications-overlay-item{cursor:pointer;font-size: 14px;margin: 14px 0 -2px;max-width: 280px;padding: 7px 28px;}
 .notifications-overlay-item:not(.nohover):hover{background:#f1f1f1;}
-.notifications-overlay-item .clearall {width: 16px;margin: 0;vertical-align: -3px;margin-right: 5px;}
\ No newline at end of file
+.notifications-overlay-item .clearall {width: 16px;margin: 0;vertical-align: -3px;margin-right: 5px;}
+.sidebar-notification {width: 26px;height: 26px;position: absolute;margin: -30px 42px;font-size: 13px;}
```

Comment:
I think L210 is a duplicate of L200, must have been something to do with the rebase.